app/src/components/parts/WalletUserList.vue

```diff
@@ -25,6 +27,9 @@ export default {
   methods: {
     click: function(wallet) {
       this.$emit('open', wallet);
+    },
+    clickDonation: function(wallet) {
+      this.$emit('openDonation', wallet);
     }
```

Comment:
同一プロジェクト内ではコード規約は統一しましょう。
今回は他では下記のように省略記法を使っていますので、そちらに合わせるのがいいかと思います。

```suggestion
    click(wallet) {
      this.$emit('open', wallet);
    },
    clickDonation(wallet) {
      this.$emit('openDonation', wallet);
    }
```